src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/PersonsTest.kt

```diff
@@ -208,7 +213,7 @@ class PersonsTest: AbstractTest() {
             assertTrue(personTotalTimes.find { it.billableProjectTime == 52 } != null)
             assertTrue(personTotalTimes.find { it.internalTime == 378 } != null)
             personTotalTimes.forEach {
-                assertTrue(it.balance < 0)
+                assertTrue(it.balance <= 0)
```

Comment:
As in a previous comment, lets discuss so you can explain to me whats going on here with this assertion, it seems suspiciously vague.